src/flags/display.rs

```diff
@@ -26,7 +26,11 @@ impl Configurable<Self> for Display {
     /// [None].
     fn from_arg_matches(matches: &ArgMatches) -> Option<Self> {
         if matches.is_present("all") {
-            Some(Self::All)
+            if matches.is_present("directory-only") {
```

Comment:
This can be better accomplished by just flipping the order of the if conditions.

Reply:
You mean to check first "directory-only" and then the if "all" nested?